gui/cache.go

```diff
@@ -382,11 +382,17 @@ func (c *Cache) getArchivedPayments(first, last int, accountID string) (int, []*
 }
 
 func (c *Cache) getArchivedPaymentsTotal(accountID string) string {
+	c.archivedPaymentsMtx.RLock()
+	defer c.archivedPaymentsMtx.RUnlock()
```

Comment:
I think the `defer` can be avoided for these.

```go
func (c *Cache) getArchivedPaymentsTotal(accountID string) string {
	c.archivedPaymentsMtx.RLock()
	total := c.archivedPaymentTotals[accountID]
	c.archivedPaymentsMtx.RUnlock()	
	return amount(total)
}

func (c *Cache) getPendingPaymentsTotal(accountID string) string {
	c.pendingPaymentsMtx.RLock()
	total := c.pendingPaymentTotals[accountID]
	c.pendingPaymentsMtx.RUnlock()
	return amount(total)
}
```

Reply:
done